crawler/vison/spiders/visonSpider.py

```diff
@@ -3,43 +3,52 @@
 from scrapy.spiders import CrawlSpider
 import psycopg2
 
+prevLinks = []
+nextLinks = []
+
 class VisonspiderSpider(scrapy.Spider):
     name = 'visonSpider'
     start_urls = ['https://www.wikipedia.org/']
 
     def parse(self, response):
         # Cursor for the db
         
-        conn = psycopg2.connect(
-            host = "127.0.0.1",
-            database = "vison",
-            user = "postgres",
-            password = "9681"
```

Comment:
Just a suggestion.. Don't include your own password! Leave it blank or just add "password"

Reply:
Actually the same credentials were provided in the previous version of the code, so I thought it would be better if I maintain the consistency. But anyway, I will make sure to take care of this in future.